src/redux/actions.js

```diff
@@ -342,6 +342,15 @@ export function resetMapReloaded() {
     };
 }
 
+export const EQUIPMENTS_INITIALIZED = 'EQUIPMENTS_INITIALIZED';
```

Comment:
You could rename all those variables and functions with the map prefix. 
For example: EQUIPMENTS_INITIALIZED to MAP_EQUIPMENTS_INITIALIZED

Reply:
done